sources/lib/datalab/datalab/context/_utils.py

```diff
@@ -19,6 +19,17 @@
 import os
 
 
+def _in_docker():
+  try:
+    for line in open('/proc/1/cgroup'):
+      if line.find(':/docker/') > 0:
+        return True
+      break
```

Comment:
Based on our earlier discussion, will change this to check for DATALAB_ENV envvar and /datalab directory.